src/modules/familyMember/application/familyMember.service.ts

```diff
@@ -0,0 +1,95 @@
+import { BadRequestException, ConflictException, Inject, Injectable, NotFoundException } from "@nestjs/common";
+import { FAMILY_MEMBER_REPOSITORY_TOKEN, IFamilyMemberRepository } from "../domain/familyMember.repository.interface";
+import { CreateFamilyMemberDTO } from "./createFamilyMember.dto";
+import { UpdateFamilyMemberDTO } from "./updateFamilyMember.dto";
+import { FamilyMemberEntity } from "../domain/familyMember.entity";
+import { PrismaService } from "src/prisma/prisma.service";
+
+@Injectable()
+export class FamilyMemberService {
+    constructor(
+        @Inject(FAMILY_MEMBER_REPOSITORY_TOKEN)
+        private readonly familyMemberRepository: IFamilyMemberRepository,
+        private readonly prisma: PrismaService,
+    ) {}
+
+    async create(dto: CreateFamilyMemberDTO): Promise<FamilyMemberEntity> {
+        if (dto.email) {
+            const existingMember = await this.familyMemberRepository.findByEmail(dto.email);
+            if (existingMember) {
+                throw new ConflictException(`Email '${dto.email}' already in use!`);
+            }
+        }
+        
+        await this.validateDependencies(dto.addressId, dto.studentIds);
```

Comment:
Olhar a sugestão que eu coloquei mais abaixo sobre o método `this.validateDependencies` .